appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -299,6 +301,13 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
         }
     }
 
+    /**
+     * Register custom view to be rendered if key matches incoming custom view in any flow
+     */
+    public fun registerCustomFrame(identifier: String, view: AppcuesComposeView) {
```

Comment:
change to 

`static fun registerCustomComponent(identifier: String, view: AppcuesCustomComponentView)``